src/main/java/us/myles/ViaVersion/ViaVersionPlugin.java

```diff
@@ -217,7 +237,10 @@ public synchronized void handleAdd(Object o) {
                         }
                     }
                 }
-            }
+
+            }.find(connection, connection.getClass(), ElementType.FIELD);
+
+            getLogger().info("Successfully injected!");
```

Comment:
This doesn't really mean successfully injected so it shouldn't be here. It might not of injected properly due to late bind, this needs removing